### PR TITLE
Add esCount functionality

### PR DESCRIPTION
**Pull request by yoitsro**
Hey! Desperately needing this functionality. I've added the necessary tests. Let me know if you have any questions. Cheers!

### CODE REVIEW

**Comment by IGZgustavomarin** [comment]
Does your tests pass locally?

**Comment by guumaster** [comment]
Does your tests pass locally?

**Comment by guumaster** [comment]
please reopen after correcting the tests

**Comment by yoitsro** [comment]
They did pass locally... Very strange.

**Comment by guumaster** [comment]
Please merge latest version, 2.1.3, and apply [this patch](https://gist.github.com/guumaster/4510d5fee4439d7df4c2), then reopen this PR.

**Comment by yoitsro** [comment]
It's now looking as though it's failing because there are 'No living connections', suggesting that the Elasticsearch instance is down or the Elasticsearch config is incorrect. Any thoughts?

**Comment by guumaster** [comment]
Please remove the file `esCount-patch.diff` and I'll merge it.

**Comment by yoitsro** [comment]
All done. Just waiting for the tests to run :)

File: test/count-test.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+CommentSchema.plugin(mongoosastic, {
+  bulk: {
+    size: 2,
+    delay: 100
+  }
+});
+
+var Comment = mongoose.model('Comment', CommentSchema);
+
+describe.only('Count', function() {
```
Comment (guumaster) [inline test/count-test.js]: Please, also remove this filter to run all tests.

**Comment by guumaster** [comment]
Thanks for the addition. If you could add a small example in the README.md it would be great.

**Pull request merged**